frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.init

```diff
@@ -79,16 +77,16 @@ start() {
   mkdir -p  "${orchestrator_cvd_artifacts_dir}"
   chown _cutf-operator:cvdnetwork "${orchestrator_cvd_artifacts_dir}"
 
-  ORCHESTRATOR_HTTP_PORT="${orchestrator_http_port}" \
-  ORCHESTRATOR_HTTPS_PORT="${orchestrator_https_port}" \
-  ORCHESTRATOR_TLS_CERT_DIR="${orchestrator_tls_cert_dir}" \
-  ORCHESTRATOR_INSTANCE_MANAGER_ENABLED="${orchestrator_instance_manager_enabled}" \
-  ORCHESTRATOR_ANDROID_BUILD_URL="${orchestrator_android_build_url}" \
-  ORCHESTRATOR_CVDBIN_ANDROID_BUILD_ID="${orchestrator_cvdbin_android_build_id}" \
-  ORCHESTRATOR_CVDBIN_ANDROID_BUILD_TARGET="${orchestrator_cvdbin_android_build_target}" \
-  ORCHESTRATOR_CVD_ARTIFACTS_DIR="${orchestrator_cvd_artifacts_dir}" \
+  set_config ORCHESTRATOR_HTTP_PORT ${orchestrator_http_port}
```

Comment:
nit: put quotes around the second parameters.
you're checking that $2 is non-empty, not that it's defined or not, you might as well ensure it's defined

Reply:
good point, done. PTAL